Networking/Networking/Mapper/CustomerMapper.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+
+/// Mapper: Customer
+///
+struct CustomerMapper: Mapper {
+
+    /// Site Identifier associated to the refund that will be parsed.
+    /// We're injecting this field via `JSONDecoder.userInfo` because SiteID is not returned in any of the Customer Endpoints.
+    ///
+    let siteID: Int64
+
+
+    /// (Attempts) to convert a dictionary into a single Customer.
+    ///
+    func map(response: Data) throws -> Customer {
+        let decoder = JSONDecoder()
+        decoder.dateDecodingStrategy = .formatted(DateFormatter.Defaults.dateTimeFormatter)
+        decoder.userInfo = [
+            .siteID: siteID
+        ]
+
+        return try decoder.decode(CustomerEnvelope.self, from: response).customer
+    }
+
+    /// (Attempts) to encode a Customer object into JSONEncoded data.
+    ///
+    func map(customer: Customer) throws -> Data {
+        let encoder = JSONEncoder()
+
+        return try encoder.encode(customer)
+    }
+}
```

Comment:
This method is not needed, since there is a way (added in another comment in this PR) for transforming a `Customer` in `Data`.

Reply:
Removed in 4a07d63.